frontend/package.json

```diff
@@ -55,8 +55,10 @@
     ]
   },
   "dependencies": {
-    "@patternfly/patternfly": "1.0.219",
-    "@patternfly/react-core": "2.4.1",
+    "@patternfly/patternfly": "2.4.1",
+    "@patternfly/react-core": "3.2.4",
+    "@patternfly/react-table": "git+https://git@github.com/priley86/react-table.git",
```

Comment:
these are POC branch deployments used for testing at the moment until [1250](https://github.com/patternfly/patternfly-react/pull/1250) is finalized.

Reply:
updated. `@patternfly/react-table` and `@patternfly/react-virtualized-extension` have been updated to their latest deployed npm versions.